test.js

```diff
@@ -614,3 +614,65 @@ test('sync writing is fully sync', (t) => {
   const data = fs.readFileSync(dest, 'utf8')
   t.equal(data, 'hello world\nsomething else\n')
 })
+
+// these tests do not run TravisCI. There seem to be not
+// enough memory there to trigger this situation.
+// Moreover they will fail on Node 6, as we cannot allocate a string this
+// big. It's considered a won't fix on Node 6, as it's deprecated.
+if (!process.env.TRAVIS_CI || process.versions.node.indexOf('6.') !== -1) {
+  test('write enormously large buffers async', (t) => {
```

Comment:
looks like one test is timing out on CI for all platforms/version - I wonder if this in some way registered then not run because of TAP transpilation?

Reply:
It was the timeout, let's see if we can unskip them.

Reply:
looks like that works